### ISSUE TITLE
[doc] 15.1  Starting or Restarting NFS Ganesha

### ISSUE THREAD

**Issue by supriti** [SUSE/doc-ses#767]
### Source Document
15.1  Starting or Restarting NFS Ganesha:

https://documentation.suse.com/ses/7/single-html/ses-admin/#ceph-nfsganesha-services

We should specify that nfs-ganesha service would start but it won't do anything useful unless user has created export-config and common-config. Only after export-config and common-config are created, user can export cephfs over nfs-ganesha.

A easier way would be to first explain how to setup the configuration file and then place this section: Starting or Restarting NFS Ganesha. Then it would be clear that user is expected to have right configuration in place to get a working NFS-Ganesha.

**Comment by mgfritch** [comment]
> We should specify that nfs-ganesha service would start but it won't do anything useful unless user has created export-config and common-config. Only after export-config and common-config are created, user can export cephfs over nfs-ganesha.
> 
> A easier way would be to first explain how to setup the configuration file and then place this section: Starting or Restarting NFS Ganesha. Then it would be clear that user is expected to have right configuration in place to get a working NFS-Ganesha.

The common config is created by cephadm during `ceph orch apply`. The ganesha service will then is then listed by the dashboard and exports can be configured there .. ?

**Comment by asettle** [comment]
@supriti ?

**Comment by supriti** [comment]
In that case, we should make it clear that user will need to setup export from dashboard to have a usable nfs-ganesha. Its not very clear from documentation right now.

**Comment by asettle** [comment]
@supriti we have an `Important` sign here: https://documentation.suse.com/ses/7/single-html/ses-admin/#dashboard-ogw with the actual docs here: https://documentation.suse.com/ses/7/single-html/ses-admin/#dashboard-ogw-enabling

Do you suggest a better placement or location?

**Comment by supriti** [comment]
https://documentation.suse.com/ses/7/single-html/ses-admin/#ceph-nfsganesha-create-export

Taking another look at doc, at the link above, we refer to file "export-1". But we never give an example of what this file should look like. The export can be created through dashboard or by creating this file, export-1.  

Here we can mention regarding dashboard and also give an example of export file.

**Comment by tbazant** [comment]
> https://documentation.suse.com/ses/7/single-html/ses-admin/#ceph-nfsganesha-create-export
> 
> Taking another look at doc, at the link above, we refer to file "export-1". But we never give an example of what this file should look like. The export can be created through dashboard or by creating this file, export-1.
> 
> Here we can mention regarding dashboard and also give an example of export file.

@supriti can you paste an example of `export-1` file together with steps to export such file using Dashboard?

**Comment by asettle** [comment]
> > https://documentation.suse.com/ses/7/single-html/ses-admin/#ceph-nfsganesha-create-export
> > Taking another look at doc, at the link above, we refer to file "export-1". But we never give an example of what this file should look like. The export can be created through dashboard or by creating this file, export-1.
> > Here we can mention regarding dashboard and also give an example of export file.
> 
> @supriti can you paste an example of `export-1` file together with steps to export such file using Dashboard?

Supriti no longer works at SUSE. However, the exampe of the export-1 file is here: https://confluence.suse.com/display/SUSEEnterpriseStorage/nfs-ganesha#nfsganesha-ExportConfig @mgfritch wrote this fairly extensive doc last year and can help with any NFS-related questions.
